ITE/6/README.adoc

```diff
@@ -0,0 +1,178 @@
+= ITE-6: Generalized link format
+:source-highlighter: pygments
+:toc: preamble
+:toclevels: 2
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| ITE
+| 6
+
+| Title
+| Generalized link format
+
+| Sponsor
+| link:https://github.com/santiagotorres[Santiago Torres]
+
+| Status
+| Draft
+
+| Type
+| Standards
+
+| Created
+| 2020-10-30
+
+|===
+
+[[abstract]]
+== Abstract
+
+This link:https://github.com/in-toto/ITE[in-toto enhancement (ITE)] defines a
+new schema for in-toto link files, which are now generally called
+"attestations." An attestation is authenticated metadata about one or more
+software artifacts. Instead of a fixed schema (`materials`, `products`,
+`command`, etc.), the attestation format allows users to define their own custom
+"predicates" with arbitrary schemas. This makes it easier and more natural to
+express any type of metadata about software. Several pre-defined predicate types
+cover the most common cases, including
+link:https://github.com/in-toto/attestation/blob/main/spec/predicates/provenance.md[provenance]
+for describing the origins of a software artifact,
+link:https://github.com/in-toto/attestation/blob/main/spec/predicates/spdx.md[spdx]
+for supporting link:https://spdx.dev[SPDX], and
+link:https://github.com/in-toto/attestation/blob/main/spec/predicates/link.md[link]
+as a drop-in replacement for the existing link schema.
+
+A future ITE will update the layout format to take advantage of this extra
+information.
+
+More broadly, this ITE moves in-toto to follow the
+link:https://github.com/slsa-framework/slsa/blob/main/controls/attestations.md[SLSA attestation model]
```

Comment:
optional: You can replace the link with https://slsa.dev/attestation-model

Reply:
Done.